tests/integration/components/build-header-test.js

```diff
@@ -75,6 +75,29 @@ test('render cron build', function (assert) {
   assert.ok(this.$().find('.build-title').text().match(/cron Just complete and utter joy/), 'displays cron before commit message');
 });
 
+
+test('render tag build', function (assert) {
+  let commit = {
+    subject: 'Just complete and utter joy',
+    branch: { name: 'v1.0.0' }
+  };
+  let build = {
+    eventType: 'push',
+    commit,
+    branch: { name: 'v1.0.0' },
+    tag: { name: 'v1.0.0' },
+    isTag: true
+  };
+
+  this.set('build', build);
+  this.render(hbs`{{build-header item=build commit=build.commit}}`);
+
+  assert.equal(this.$().find('.build-status svg title').text(), 'Tag');
+  assert.equal(this.$().find('.build-title .commit-branch').text().trim(), 'v1.0.0', 'displays tag name in title');
+  assert.ok(this.$().find('.commit-branch-url').text().match(/Tag v1.0.0/), 'displays link to tag on GH');
+  assert.ok(this.$().find('.commit-branch-url').attr('href').match(/releases\/tag\/v1.0.0/), 'url is correct');
```

Comment:
This makes me wonder, are there any limitations on what characters tags can contain? [This page](https://git-scm.com/docs/git-check-ref-format) purportedly lists what’s okay in a tag name, is it possible that someone can generate a tag name that needs escaping when linking? 🤔

Reply:
good point, I didn't think of that. But I guess GitHub would already make sure it's an url-able tag name?

Reply:
yeah, seems like it! I have never made a tag in my whole life haha so I’m like 🤔